app/Http/Controllers/Api/V1/TeamController.php

```diff
@@ -482,13 +494,20 @@ public function addProjects(TeamAddProjectRequest $addProjectRequest, Team $team
                 // $team->projects()->attach($project);
                 // $assigned_projects[] = $project;
                 // pushed cloning of project in background
-                CloneProject::dispatch($auth_user, $project,  $addProjectRequest->bearerToken(), $team->organization->id, $team)->delay(now()->addSecond());
+                CloneProject::dispatch($auth_user,
+                                       $project,
+                                       $addProjectRequest->bearerToken(),
+                                       $team->organization->id, $team
+                                       )
+                            ->delay(now()->addSecond());
             }
         }
         // $this->teamRepository->setTeamProjectUser($team, $assigned_projects, []);
 
         return response([
-            'message' => "Your request to add [$project->name] project in team has been received and is being processed. You will receive an email notice as soon as it is available.",
+            'message' =>
```

Comment:
Please revert it back to previous, if it makes horizontal scroll then only we use break line